packages/create-react-app/createReactApp.js

```diff
@@ -507,7 +516,27 @@ function getInstallPackage(version, originalDirectory) {
       packageToInstall = version;
     }
   }
-  return packageToInstall;
+
+  if (packageToInstall.startsWith('react-scripts-ts')) {
```

Comment:
I imagine we will have more scripts versions we want to warn about in the future. I was thinking we should have a list of them and then check against that. Something like this:

```js
const dangerousScripts = [
  {
    name: 'react-scripts-ts',
    message: 'The react-scripts-ts package is deprecated...'
  }
];
```

Reply:
Updated to support multiple scripts